### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Alhajideen**
## Math Magician
In this milestone, I was able to ;
- Create a Calculator.js file
- Design a simple calculator ui.
- Render the calculator in app.js
- Use professional gitflow.
I await your feedback as usual 😄 .

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (AmaduKamara) [inline src/App.js]: - [x] Kindly make your App.js component a class-based component. It's a requirement that these components be class-based components. Especially the Calculator and App components should be class-based for now 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Alhajideen,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ Linter checks passed 
✅ Good descriptive PR 👍 
✅ Your app is nicely styled as expected👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (Alhajideen) [reply src/App.js]: Thank you. Done now @AmaduKamara

**Review by FrederickMih — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Alhajideen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- Great job for making the App component class-based :+1: 
- No Linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**